### PR TITLE
Plans: overview & compare into single page

### PR DESCRIPTION
**Pull request by adambbecker**
[plans_new](https://cloud.githubusercontent.com/assets/1427136/15692620/6ef5b45c-275d-11e6-895f-63de220aadf9.png)

This is a very much in progress PR just meant for @rralian & @lamosty to get a headstart on some potential upcoming changes for the plans page. Most notably, it combines `/plans` & `/plans/compare` pages into a single page (`/plans`) that is more in tune with the current logged out plans views.

This PR's main purpose is just to help out with some styling suggestions as we move towards implementing the redesign.

--

_Note:_ 

Everything in this PR is suffixed with either `Redesign` (exports & imports), `-redesign` (filenames), or `--redesign` (classNames & css) in order to keep the work encapsulated.

### CODE REVIEW

File: client/components/faq/index.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/**
```
Comment (mtias) [inline client/components/faq/index.jsx]: Is this going to be a new component in devdocs?

Comment (adambbecker) [inline client/components/faq/index.jsx]: @mtias: That was the idea yeah.

File: client/components/faq/index.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import i18n from 'lib/mixins/i18n';
```
Comment (Tug) [inline client/components/faq/index.jsx]: Hi there, i18n usage has just changed recently. please use:
```
import i18n from 'i18n-calypso';
```
instead.

**Comment by lamosty** [comment]
Closing this in favor of other already merged plans redesign PRs.

**Pull request closed without merging**